docs/reference/api/connectivity/bluetooth/SecurityManager.md

```diff
@@ -1,10 +1,10 @@
 ## SecurityManager
 
-SecurityManager deals with authentication and encryption for the Bluetooth Low Energy link. The process of pairing and optionally bonding provides this. The SecurityManager achieves bonding by saving the pairing information and reusing it on subsequent reconnections in order to save time by not having to perform pairing again.
+SecurityManager deals with authentication and encryption for the Bluetooth Low Energy link. The pairing and optionally bonding processes provide this. The SecurityManager achieves bonding by saving the pairing information and reusing it on subsequent reconnections. This saves time because the pairing does not have to be performed again.
```

Comment:
Pairing always happens. Bonding may happen. Bonding means: saving pairing information for later use so that you don't have to pair again. There is no extra communication between the devices. One device may just decide to "bond" with the other which means nothing else than just save the result of the pairing process which otherwise would have to be redone on subsequent reconnections.

Reply:
Ok this is what I thought - they are different processes but related. This makes sense, thanks!